saleor/attribute/migrations/0023_lstrip_slash_from_value_file_url.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.16 on 2022-10-12 08:35
+
+from django.db import migrations
+
+BATCH_SIZE = 10000
+
+
+def update_file_urls(apps, schema_editor):
+    AttributeValue = apps.get_model("attribute", "AttributeValue")
+    queryset = AttributeValue.objects.filter(file_url__startswith="/").order_by("pk")
+    for values in queryset_in_batches(queryset):
+        for value in values:
+            value.file_url = value.file_url.lstrip("/")
+
+        AttributeValue.objects.bulk_update(values, ["file_url"])
```

Comment:
On my local dataset (145k values) this takes 125 seconds
Instead we could go with
```
AttributeValue.objects.filter(values__in=pks).update(file_url=Func(F('file_url'), Value('/'), function='ltrim'))
```
This takes below 60 seconds to complete (`queryset_in_batches` would have to yield `pks` instead of `qs`) and from what I've checked, it works like it should. From postgres docs on LTRIM: 

_Removes the longest string containing only characters in characters (a space by default) from the start of string._